cookbooks/cdo-apps/test/integration/default/serverspec/ruby_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'serverspec'
```

Comment:
Yeah, this is a barebones-simple test for now just to get started. I'll add to this (including comments) in a further set of changes that I'll be putting up in a follow-on PR.